src/openrct2/ride/Vehicle.h

```diff
@@ -376,6 +376,18 @@ struct train_ref
     Vehicle* tail;
 };
 
+namespace MiniGolfFlag
+{
+    constexpr uint8_t Flag0 = (1 << 0);
+    constexpr uint8_t Flag1 = (1 << 1);
+    constexpr uint8_t Flag2 = (1 << 2);
+    constexpr uint8_t Flag3 = (1 << 3);
+    constexpr uint8_t Flag4 = (1 << 4);
+    constexpr uint8_t Flag5 = (1 << 5); // transitioning between hole
+    constexpr uint8_t Flag6 = (1 << 6);
+    constexpr uint8_t Flag7 = (1 << 7);
```

Comment:
On https://github.com/OpenRCT2/OpenRCT2/pull/15072/commits/08632500183b3473dc3144c9eef77cb2543499b2 these two flags are unused

Reply:
yes realise that. Added it just in case can be removed now.